reference/tools/data-migration/features/online-ddl-scheme.md

```diff
@@ -91,28 +91,30 @@ DM 在同步过程中会把上述 table 分成 3 类：
 
 ### DM 对于 **online-ddl-scheme: gh-ost** 的处理
 
-- 不执行 `_test4_ghc` 的创建操作。
-- 不执行 `_test4_gho` 的创建操作，根据 ghost_schema、ghost_table 以及 dm_worker 的 server_id 删除下游 dm_meta.{task_name}\_onlineddl 的记录，清理内存中的相关信息。
+1. 不执行 `_test4_ghc` 的创建操作。
```

Comment:
Are the handling methods in sequence?

Reply:
The author said that these methods match the order of the SQL statements above, so he suggested using numbered lists. https://github.com/pingcap/docs-cn/pull/2425#discussion_r393470680

Reply:
What about giving a handling method immediately after each statement? I think it would be more user-friendly.